exercises/concept/little-sisters-vocab/.docs/introduction.md

```diff
@@ -0,0 +1,203 @@
+# Introduction
+
+A `str` in Python is an [immutable sequence][text sequence] of [Unicode code points][unicode code points].
+These could include letters, diacritical marks, positioning characters, numbers, currecy symbols, emoji, punctuation, space and line break characters, and more.
+Being immutable, a `str` object's value in memory doesn't change; methods that appear to modify a string return a new copy or instance of `str`.
+
+A `str` literal can be declared via single `'` or double `"` quotes. The escape `\` character is available as needed.
+
+```python
+
+>>> single_quoted = 'These allow "double quoting" without "escape" characters.'
+
+>>> double_quoted = "These allow embedded 'single quoting', so you don't have to use an 'escape' character".
```

Comment:
Might be worthwhile to show some sort of example with an escape character

Reply:
For completeness, it would.  It is ... sorta discouraged, since there are double AND single quotes available, which cover 95% of cases where you'd use an escape.  But if I mention it, I should demonstrate it.